inlong-sort-standalone/sort-standalone-common/src/main/java/org/apache/inlong/sort/standalone/config/holder/v2/SortConfigHolder.java

```diff
@@ -133,4 +153,15 @@ public static SortTaskConfig getTaskConfig(String sortTaskName) {
         }
         return null;
     }
+
+    public static String getAuditTag(
```

Comment:
Seems this method is never used.

Reply:
It will be used in the customized plugins of metrics reporter